test/annotation_layer_builder_overrides.css

```diff
@@ -48,3 +47,7 @@
   margin: 0;
   padding: 0;
 }
+
+.annotationLayer .popupTriggerArea {
+  outline: dashed 3px rgba(255, 0, 0, 0.2);
```

Comment:
Looking at the test-results, this looks quite similar to annotations with dashed borders.

Hence my question: Should we perhaps use the same kind of styling as for linkAnnotations (above in this file) instead here?